coveralls.json

```diff
@@ -3,7 +3,8 @@
     "minimum_coverage": 87
   },
   "skip_files": [
-    "test/support"
+    "test/support",
+    "lib/course_planner/notifier.ex"
```

Comment:
Can we mark as not relevant in some way? I isolated the hardly testable part of the code to this file and I believe it shouldn't be treated as "missing coverage" when it's not coveralble (or at least it's hard to).